modulemd/meson.build

```diff
@@ -71,6 +71,11 @@ install_headers(
   subdir : header_path,
 )
 
+modulemd_dep = declare_dependency(
+    include_directories : include_directories('.'),
+    link_with : modulemd_lib,
+)
```

Comment:
it seems you need also add `dependencies: [gobject]`... Btw, also you should be able to change all references later to `dependencies:[modulemd_dep]` instead of `link_with:modulemd_lib,dependencies:[gobject]`

Reply:
If it needs gobject, it probably needs yaml as well, yes?

Reply:
you don't expose libyaml in headers, so no.